build/Containerfile.sidecar

```diff
@@ -7,16 +7,14 @@ ADD . /workspace/go/src/github.com/csi-addons/kubernetes-csi-addons
 ENV GOPATH=/workspace/go
 WORKDIR /workspace/go/src/github.com/csi-addons/kubernetes-csi-addons
 
-# Build
-RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor -a -o /workspace/csi-addons-sidecar ./sidecar
-
-# Build the csi-addons tool for admin usage and testing
-RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor -a -o /workspace/csi-addons ./cmd/csi-addons
+# Build the sidecar and csi-addons tool for admin usage and testing
+RUN make build
 
 # Use distroless as minimal base image to package the sidecar binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM gcr.io/distroless/static:latest
 WORKDIR /
-COPY --from=builder /workspace /usr/bin/
+COPY --from=builder /workspace/go/src/github.com/csi-addons/kubernetes-csi-addons/bin/csi-addons-sidecar /usr/sbin/
+COPY --from=builder /workspace/go/src/github.com/csi-addons/kubernetes-csi-addons/bin/csi-addons /usr/bin/
```

Comment:
it should be copied to `/usr/sbin`?

Reply:
`/usr/sbin` is the path for executables that act like servers or are for admin users. `csi-addons` is a tool that can be used by any user to talk to the csi-driver. I though it would be more suitable in `/usr/bin`. The distinction between the paths is not very important to me, I am happy yo change either, both should be in the `$PATH` when an admin user logs in the pod.